infrastructure/application/index.ts

```diff
@@ -260,6 +260,7 @@ export = async () => {
     external: true,
     vpc,
     subnets: networking.requireOutput("publicSubnetIds"),
+    idleTimeout: 120,
```

Comment:
question: since this error is quite specific so far, should we maybe think about route-specific timeout bumps instead of our overall server? 

admittedly not super clear on when to update config in pulumi versus express though - but was thinking something like `server.setTimeout(60 * 1000)` overall then `req.setTimeout(120 * 1000)` directly in applicable endpoints? i'd guess that this `idleTimeout` still needs to match our highest possible route timeout.

appreciate this is just intended to be a quick interim fix though so trust your judgement here, thanks for opening something today :+1:

Reply:
No this is a good shout! 👌 

You're right that this would need to match our highest value so that sounds like a much more sensible and safe approach - I'll try and implement that just now.